Pei_HW/session03/wsgi/calculator.py

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env python3
+def add (x, y):
+    return x + y
+def subtract (x, y):
+    return x - y
+def multiply (x, y):
+    return x * y
+def divide (x, y):
+    return x / y
+
+def page():
+    front_page = """<html>
+    <head>
+    <title>Simple Calculator</title>
+    </head>
+
+    <body>
+    <p>This simple calculator can perform the following math related operations:</p>
+    <p>Addition</p>
+    <p>ex: http://localhost:8080/add/5/3</p>
+    <p>Subtraction</p>
+    <p>ex: http://localhost:8080/subtract/5/3</p>
+    <p>Multiplication</p>
+    <p>ex: http://localhost:8080/multiply/5/3</p>
+    <p>Division</p>
+    <p>ex: http://localhost:8080/divide/5/3</p>
+    </body>
+    </html>"""
+    return front_page
+
+def calculate (operator, operands):
+    answer_page = """<html>
+    <head>
+    <title>Simple Calculator</title>
+    </head>
+    <body>
+    Answer: {}
+    </body>
+    </html>"""
+    if operator == 'add':
+        answer = add(*operands)
+    elif operator == 'subtract':
+        answer = subtract (*operands)
+    elif operator == 'multiply':
+        answer = multiply(*operands)
+    elif operator == 'divide':
+        answer = divide(*operands)

```

Comment:
These functions are all already available in the operator module. https://docs.python.org/3.5/library/operator.html
Also, try to think of a way to do this that eliminates the need for this if statement